src/drawscene.h

```diff
@@ -25,15 +25,13 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "minimap.h"
 #include "irrlichttypes_extrabloated.h"
 
-
 void draw_load_screen(const std::wstring &text, IrrlichtDevice *device,
 		gui::IGUIEnvironment *guienv, float dtime = 0, int percent = 0,
 		bool clouds = true);
 
-void draw_scene(video::IVideoDriver *driver, scene::ISceneManager *smgr,
-		Camera &camera, Client &client, LocalPlayer *player,
-		Hud &hud, Minimap &mapper, gui::IGUIEnvironment *guienv,
-		const v2u32 &screensize, const video::SColor &skycolor,
-		bool show_hud, bool show_minimap);
+void draw_scene(video::IVideoDriver *driver, scene::ISceneManager *smgr, Camera &camera,
+		Client &client, LocalPlayer *player, Hud &hud, Minimap &mapper,
+		gui::IGUIEnvironment *guienv, const v2u32 &screensize,
+		const video::SColor &skycolor, bool show_hud, bool show_minimap);
```

Comment:
both are compact, one respect 80 chars other 90 chars are you sure you want me to revert this change ?